it/api/endpoints/lossClaim/list/v3/ListLossClaimsControllerISpec.scala

```diff
@@ -312,16 +415,90 @@ class ListLossClaimsControllerISpec extends V3IntegrationBaseSpec {
       }
     }
 
+    "querying for a TYS specific taxYear and businessId" in new NonTysTest {
```

Comment:
I don't think we need to test for the TYS scenario for every integration test, as it is the same controller so I think as long as we have one then it will suffice.